refinery/tool_manager/tasks.py

```diff
@@ -14,6 +14,6 @@ def django_docker_cleanup():
 
 @task()
 def start_container(visualization_tool):
-    visualization_tool.django_docker_client.run(
+    return visualization_tool.django_docker_client.run(
```

Comment:
It's best to avoid returning a result in a task: http://docs.celeryproject.org/en/3.1/userguide/tasks.html#ignore-results-you-don-t-want